bindings/rascal/models/IP_ase_interface.py

```diff
@@ -41,3 +42,14 @@ def calculate(self, atoms=None, properties=['energy'],
         self.results['energy'] = energy
         self.results['free_energy'] = energy
         self.results['forces'] = forces
+
+    def _get_init_params(self):
+        init_params = dict(model=self.model, representation=self.representation)
+        init_params.update(**self.kwargs)
+        return init_params
+
+    def _set_data(self, data):
+        pass
```

Comment:
I would even suggest it here; maybe in the future we will want to implement default behaviour for `_set_data()`.

```suggestion
        super()._set_data(data)
```